waspc/cli/src/Wasp/Cli/Command/BashCompletion.hs

```diff
@@ -46,30 +43,28 @@ bashCompletion = do
         "test",
         "studio"
       ]
-    dbSubCommands = ["migrate-dev", "studio"]
+    dbSubCommands = ["start", "reset", "seed", "migrate-dev", "studio"]
     listMatchingCommands :: String -> [String] -> Command ()
     listMatchingCommands cmdPrefix cmdList = listCommands $ filter (cmdPrefix `isPrefixOf`) cmdList
     listCommands :: [String] -> Command ()
     listCommands cmdList = liftIO . putStrLn $ unlines cmdList
 
--- generate the bash completion script
-generateBashCompletionScript :: Command ()
-generateBashCompletionScript =
-  liftIO $ getDataFileName "Cli/bash-completion" >>= readFile >>= putStr
-
 -- return the bash completion instruction
 printBashCompletionInstruction :: Command ()
 printBashCompletionInstruction =
   liftIO . putStrLn $
     unlines
-      [ "Run the following command to generate bash completion script for wasp on your machine:",
+      [ "Setting up Bash auto-completion for Wasp",
+        "",
+        "1. Configure your shell to auto-complete Wasp commands:",
+        "   - Bash (default for most Linux and WSL): edit ~/.bashrc",
+        "   - Zsh (default for macOS): edit ~/.zshrc",
```

Comment:
Maybe emphasize that these are the default locations since an increasing number of distros have started respecting the XDG config standard.